dataapiv1/spec.yaml

```diff
@@ -3,53 +3,53 @@ info:
   version: 1.0.0
   title: Minimal Couchbase Data API
 paths:
-  '/v1/buckets/{bucket_name}/scopes/{scope_name}/collections/{collection_name}/docs/{doc_key}':
+  '/v1/buckets/{bucketName}/scopes/{scopeName}/collections/{collectionName}/documents/{docKey}':
     parameters:
-      - name: Authorization
-        in: header
-        schema:
-          type: string
-        required: true
+      - in: header
+        name: Authorization
         description: Header for authentication
-      - name: bucket_name
-        in: path
         schema:
           type: string
         required: true
+      - in: path
+        name: bucketName
         description: name of the bucket containing the document
-      - name: scope_name
-        in: path
         schema:
           type: string
         required: true
+      - in: path
+        name: scopeName
         description: name of the scope containing the document
-      - name: collection_name
-        in: path
         schema:
           type: string
         required: true
+      - in: path
+        name: collectionName
         description: name of the collection containing the document
-      - name: doc_key
-        in: path
         schema:
           type: string
         required: true
+      - in: path
+        name: docKey
         description: name of the document
+        schema:
+          type: string
+        required: true
     get:
       operationId: getDocument
       tags:
         - Single Document Operations
       parameters:
-        - name: Accept-Encoding
-          in: header
+        - in: header
+          name: Accept-Encoding
           description: The encoding of the document
           schema:
             type: array
             items:
               $ref: '#/components/schemas/DocumentEncoding'
-        - name: project
-          in: query
-          description: fields to project from the document
+        - in: query
+          name: project
```

Comment:
"Projection" is the commonly known term for this type of feature, and "project" is the way it's typically exposed.  This is also the name that's already used throughout the SDKs for the same functionality.  I don't think we should change the name when the context for confusion here is quite far removed from any confusion with a Capella project, especially since in Capella it would be specified as a ProjectId or ProjectName.